vision/api/label/snippets.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
2016.

Also, I don't think the header on go/releasing/preparing includes "All Rights Reserved" anymore.